code-challenges.test.js

```diff
@@ -10,53 +10,125 @@
+    if (number === 0){
+      return "you cannot divide by zero"
+    } else if(number % 3 === 0 ){
+    return " is divisible by 3"
+     } else {
+        return "is not divisible by three"
+    }
+  }
 const num1 = 15
-// Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
-
-
-// b) Create the function that makes the test pass.
+console.log(numberDivisibleBy3(num1))
+console.log(numberDivisibleBy3(num2))
+console.log(numberDivisibleBy3(num3))
 
 
-
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+// // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("capitalString", () => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+      const randomNouns1 = []
+      const randomNouns2 = []
+      expect(capitalString(randomNouns1))
+      expect(capitalString(randomNouns2))
+    })
+  })
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
 
+//PseudoCode
+  //create function to map new array with the first letter of each word capitalized
+    // use .map to evaluate the array
+        //use .map to create new array by returning the 1st character of each word with .toUpperCase()
+        // concat that return value with remainder of string.
 
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+const capitalString = (string) => {
+    return string.map(value=> {
+      return value.charAt(0).toUpperCase() + value.substring(1);
+    })
+  }
 
-// a) Create a test with expect statements for each of the variables provided.
+console.log(capitalString(randomNouns1))
+
+
+// // b) Create the function that makes the test pass.
+
+// // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+describe("firstVowel", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+      const vowelTester1 = []
+      const vowelTester2 = []
+      const vowelTester3 = []
```

Comment:
These variables are empty.  This is a great place to place your test variables, just make sure they have data that you wish to test in them.